spec/commands/v2/discard_draft_spec.rb

```diff
@@ -8,9 +8,11 @@
     end
 
     let(:expected_content_store_payload) { { base_path: "/vat-rates" } }
+    let(:content_id) { SecureRandom.uuid }
```

Comment:
Since `content_id` is not actually used in the file anywhere on its own, we can just leave it to FactoryGirl to create one for our document.